frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -205,9 +205,9 @@ function roleSelected(evt: any) {
                 <form id="grantAccessForm" class="form-container">
                     <div class="row">
                         <div class="form-group col-md-3 px-0">
-                            <label for="domainInput" class="control-label"
-                                >Select user's domain</label
-                            >
+                            <p class="control-label-radio">
+                                Select user's domain
+                            </p>
```

Comment:
May it's not related to this ticket issue, but what's the main reason to change from "\<label>" tag to "\<p>" tag with custom class? Why not "\<div>" or "\<span>" tag, just curious.

Also is the intention for newly created custom css "**control-label-radio**" class only for this component?

Reply:
Hello, Ian! Thanks for the review!
Well, it can be a div or a span. Probably make more sense to use a span and use just one class.

Reply:
In this case for trying to resolve radios' group title, your previous change make sense except on using <p> tag for me.
But Catherine has adjust this and leave a comment. Please see her reasoning.
I feel it is acceptable. So I will mark this as resolved.
Later if we face the same and feel it is annoying, we can discuss and create a custom class again.